doc/classes/RichTextLabel.xml

```diff
@@ -551,6 +551,9 @@
 		<member name="deselect_on_focus_loss_enabled" type="bool" setter="set_deselect_on_focus_loss_enabled" getter="is_deselect_on_focus_loss_enabled" default="true">
 			If [code]true[/code], the selected text will be deselected when focus is lost.
 		</member>
+		<member name="drag_and_drop_selection_enabled" type="bool" setter="set_drag_and_drop_selection_enabled" getter="is_drag_and_drop_selection_enabled" default="true">
+			If [code]true[/code], allow dragging of selected text.
```

Comment:
I think the description should be the same as in other nodes, as it refers to the drag operation alone. You can't drop to RichTextLabel, but `drag_and_drop_selection_enabled` does not disable dropping in LineEdit/TextEdit.